Changelog

```diff
@@ -1,8 +1,69 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.2.0
+    - FEATURE: Enforce C++14 requirement (Chocobo1)
```

Comment:
@sledgehammer999:  For better accuracy, this particular change should probably say something like: 

> OTHER: Raise minimum C++ version to C++14 (Chocobo1)